numpy/polynomial/chebyshev.py

```diff
@@ -1058,7 +1057,6 @@ def chebint(c, m=1, k=[], lbnd=0, scl=1, axis=0):
             if n > 1:
                 tmp[2] = c[1]/4
             for j in range(2, n):
-                t = c[j]/(2*j + 1)  # FIXME: t never used
```

Comment:
@charris do you know if there is a reason this is a FIXME comment and was not deleted earlier?

Reply:
No, it was probably noted in passing in another PR... #12448, another cleanup. I suspect the author didn't have enough confidence to nuke it.